cmd/cdi-apiserver/apiserver.go

```diff
@@ -119,6 +121,15 @@ func main() {
 
 	snapClient := snapclient.NewForConfigOrDie(cfg)
 
+	mgr, err := manager.New(cfg, manager.Options{})
```

Comment:
sure you want to create a manager just for the sake of client? @arnongilboa is initializing a cached cr client in same app in https://github.com/kubevirt/containerized-data-importer/pull/2813/files#diff-1189f871222be9b4a380b421d75ed238831eca4554a236bfa61f15247e25ac99R128

Reply:
Yeah I don't think there is any additional overhead using a manager and it seems cleaner to me

Reply:
To me, the manager is the entrypoint for the Controller Runtime API.  And last time I went through the manager code, I don't recall it doing much other than creating the client and starting controllers.  So if there are no controllers I don't see any issues.  Are you aware of any unnecessary work/data being created?

Reply:
At the very least I recall it creates a metrics listener, anyway, I may be the only one finding this weird
cc @awels wdyt

Reply:
Okay, if metrics server is started, I'll change to create client directly